src/releasers/go-yoshi.ts

```diff
@@ -176,7 +176,7 @@ export class GoYoshi extends ReleasePR {
       changelogEntry,
       updates,
       version: candidate.version,
-      includePackageName: this.monorepoTags,
+      includePackageName: false,
```

Comment:
go is unique in that the top level release currently represents a "non-mono-repo" release, and submodules represent releases for individual libraries.